### PR TITLE
Feat 1237/stamp detail

### PR DESCRIPTION
**Pull request by 0xZakk**
Static version of the stamp detail page (still not pulling from the database)

### CODE REVIEW

File: packages/app/components/StampHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  padding: "0.75rem 2rem",
+});
+
+const Flex = styled("div", {
+  display: "flex",
+});
+
+const Container = styled("div");
+
+const StampHeader = ({ name, symbol, image, supply }) => {
```
Comment (dvargasfuertes) [inline packages/app/components/StampHeader.tsx]: Deployment is failing due to these props not being typed. Could we add types?

**Review by dvargasfuertes — COMMENTED** [review]
Let's get the deployment passing so that we could poke around on the branched version

**Comment by 0xZakk** [comment]
@dvargas92495 should be good now

**Review by dvargasfuertes — COMMENTED** [review]
Looks good! We should keep it on the branch until we reconnect data + actions. Should I pick it up from here or do you still have it?

**Comment by 0xZakk** [comment]
@dvargas92495 Can you pick this one up from here?

**Pull request merged**